app/templates/cli.js

```diff
@@ -1,9 +1,13 @@
 #!/usr/bin/env node
-'use strict';
+'use strict';<% if (babel) { %>
+import meow from 'meow';
+import <%= camelModuleName %> from './';
+
+const cli = meow([<% } else { %>
 var meow = require('meow');
 var <%= camelModuleName %> = require('./');
 
-var cli = meow([
+var cli = meow([<% } %>
```

Comment:
Ahh. I see. He's using [`redent`](https://github.com/sindresorhus/redent) [here](https://github.com/sindresorhus/meow/blob/master/index.js#L54) to allow that. Cool.